### PR TITLE
Disable right click menu in Studio

### PR DESCRIPTION
**Pull request by chrisplatorres**
The PR returns false oncontextmenu for launcher's index.html which disables the annoying right click menu.
  
rightclickCapture

### CODE REVIEW

**Comment by modulesio** [comment]
This looks good, but is it possible to put this into `App.js` and bind it with React?

Going forward we'll probably want a custom context menu there rather than a global hijack.

File: examples/launcher/src/App.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import './css/App.css';
 import Engine from './components/Engine.jsx'
 
+document.oncontextmenu = function() {
```
Comment (modulesio) [inline examples/launcher/src/App.js]: I was thinking `onContextMenu={() => false}` in the JSX.

We just need to test whether that works as advertised.

Comment (chrisplatorres) [reply examples/launcher/src/App.js]: Testing now.

Comment (chrisplatorres) [reply examples/launcher/src/App.js]: moved in cfe0616

**Pull request merged**